pkg/istio-agent/xds_proxy.go

```diff
@@ -449,10 +409,55 @@ func (p *XdsProxy) HandleUpstream(ctx context.Context, con *ProxyConnection, xds
 }
 
 func (p *XdsProxy) handleUpstreamRequest(con *ProxyConnection) {
+	// Handle downstream xds
```

Comment:
Having a function named `handleUpstreamRequest` that does `Handle downstream xds` seems misleading?

Reply:
maybe should recomment it